compiler/luci/service/src/CircleShapeInferenceRule.test.cpp

```diff
@@ -38,14 +35,8 @@ namespace
 
 bool shape_pass(loco::Graph *g)
 {
-  loco::CanonicalShapeInferenceRule canonical_rule;
-  luci::CircleShapeInferenceRule circle_rule;
-  loco::MultiDialectShapeInferenceRule rules;
-
-  rules.bind(loco::CanonicalDialect::get(), &canonical_rule)
-    .bind(luci::CircleDialect::get(), &circle_rule);
-
-  return loco::apply(&rules).to(g);
+  luci::CircleShapeInferencePass shape_pass;
+  return shape_pass.run(g);
```

Comment:
As we don't use `loco::apply(&rules).to(g);` anymore we have to make same traverseral order.
You may have to copy `CircleShapeInferencePass` inside here to test exact thing as `CircleShapeInferencePass` does: because of depenceny.

This helper was to forget which Pass to test for each cases, one or several.
So, we can remove `shape_pass` and call each pass for each test.

Or, introduce another sub module that builds before pass and service so that both can link.
I can't think of what to name this... I remember I had this trouble when worked on this...

Reply:
If both of them is fine, I prefer to call each pass for each test...!
I will apply the former suggestion :)